include/openssl/kdf.h

```diff
@@ -29,6 +29,9 @@ extern "C" {
 # define EVP_PKEY_CTRL_SCRYPT_R                 (EVP_PKEY_ALG_CTRL + 11)
 # define EVP_PKEY_CTRL_SCRYPT_P                 (EVP_PKEY_ALG_CTRL + 12)
 # define EVP_PKEY_CTRL_SCRYPT_MAXMEM_BYTES      (EVP_PKEY_ALG_CTRL + 13)
+# define EVP_PKEY_CTRL_PBKDF2_MD                (EVP_PKEY_ALG_CTRL + 14)
+# define EVP_PKEY_CTRL_PBKDF2_SALT              (EVP_PKEY_ALG_CTRL + 15)
+# define EVP_PKEY_CTRL_PBKDF2_ITER              (EVP_PKEY_ALG_CTRL + 16)
```

Comment:
For similar reasons, I can see a future where other KDFs also have an iteration count parameter, so it might be appropriate to give a more generic name (i.e. remove `_PBKDF2`)